config/bin_version_strings.cfg

```diff
@@ -119,11 +124,12 @@ glibc::"GNU\ C\ Library\ \(.*\)\ stable\ release\ version\ [0-9]\.[0-9]+,\ "
 gnu_cpio::"\(GNU\ cpio\)\ [0-9]\.[0-9]+$"
 gnu_findutils::"\(GNU\ findutils\)\ [0-9]\.[0-9]+\.[0-9]+$"
 gnu_findutils::"\(GNU\ findutils\)\ [0-9]\.[0-9]+\.[0-9]+-[a-z]+$"
-gnu_gettext-runtime::"\(GNU\ gettext-runtime\)\ [0-9]\.[0-9]+\.[0-9]+\.[0-9]+$"
+gnu_gettext-runtime::"\(GNU\ gettext-runtime\)\ [0-9]\.[0-9]+\.[0-9]+"
```

Comment:
as far as I can see we will loose the detection of the following version from the fritzbox firmware:
[+] Found version details:      gnu gettext-runtime     :       0.19.8.1        :       CVEs: 0         :       Exploits: 0  
could you please leave both in it and add a "$" to your new detection regex?

Reply:
done